DashAI/back/api/api_v1/endpoints/datasets.py

```diff
@@ -239,14 +241,21 @@ async def upload_dataset(
         dataset = to_dashai_dataset(dataset)
 
         if not parsed_params.splits_in_folders:
-            dataset = dataloader.split_dataset(
-                dataset,
+            n = len(dataset["train"])
+            train_indices, test_indices, val_indices = split_indices(
```

Comment:
Prefer indexes over indices:

```suggestion
            train_indexes, test_indexes, val_indexes = split_indexes(
```